File: study/computer-science/data-structure-and-algorithms/code/practice/miscellaneous/FindTheNumberOfPower.cpp
// CPP Program to calculate
// no. of digits  in a^b
#include <iostream>
#include <math.h>
using namespace std;

//  function to calculate number
//  of digits in a^b
int no_of_digit(int a, int b){
    return ((int)(b * log10(a))+1)
}

//  driver program

int main(){
    int a = 2, b = 100;
    cout << "no. of digits =  "<<
                    no_of_digit(a,b);
}

// By Smitha

// ISSUE: Cannot debug for redirected to the Setting tab after pressing F5

File: study/computer-science/data-structure-and-algorithms/code/practice/miscellaneous/FindTheNumberOfPower.cs
// C# Program to calculate
// no. of digits in a^b
using System;

class fuck{

    // functions to calculate number
    // of digits in a^b
    static int no_of_digit(int a, int b){
        return ((int)(b * Math.Log10(a)) + 1);
    }

    // driver program
    public static void Main(){
        int a = 2, b = 100;
        Console.Write("no. of digits = " +
                        no_of_digit(a,b));
    }
}


// By Smitha

// ISSUE: Can't work with VSC